packages/combobox/src/Chip.tsx

```diff
@@ -0,0 +1,198 @@
+
+      // TODO - refine these styles
+      /* &:focus, */
+      &:focus-within {
+        background-color: var(--lg-combobox-chip-focus-color);
+      }
+    `,
+  );
+};
+
+const chipText = css`
+  padding-inline: var(--lg-combobox-chip-padding-x);
+`;
+
+const chipButton = css`
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+  outline: none;
+  border: none;
+  background-color: transparent;
+  color: var(--lg-combobox-chip-icon-color);
+  cursor: pointer;
+
+  &:before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 1px;
+    background-color: var(--lg-combobox-chip-hover-color);
+  }
+
+  &:hover {
+    background-color: var(--lg-combobox-chip-hover-color);
+  }
+`;
+
+export const Chip = React.forwardRef<HTMLSpanElement, ChipProps>(
+  ({ displayName, isFocused, onRemove }: ChipProps, forwardedRef) => {
+    const {
+      darkMode,
+      size,
+      chipTruncationLocation,
+      chipCharacterLimit = 12,
+    } = useContext(ComboboxContext);
+
+    const isTruncated =
+      !!chipCharacterLimit &&
+      !!chipTruncationLocation &&
+      chipTruncationLocation !== 'none' &&
+      displayName.length > chipCharacterLimit;
+
+    const buttonRef = useRef<HTMLButtonElement>(null);
+
+    const truncatedName = useMemo(() => {
+      if (isTruncated) {
+        const ellipsis = '…';
+        const chars = chipCharacterLimit - 4;
+
+        switch (chipTruncationLocation) {
+          case 'start': {
+            const end = displayName
+              .substring(displayName.length - chars)
+              .trim();
+            return ellipsis + end;
+          }
+
+          case 'middle': {
+            const start = displayName.substring(0, chars / 2).trim();
+            const end = displayName
+              .substring(displayName.length - chars / 2)
+              .trim();
+            return start + ellipsis + end;
+          }
+
+          case 'end': {
+            const start = displayName.substring(0, chars).trim();
+            return start + ellipsis;
+          }
+
+          default: {
+            return displayName;
+          }
+        }
+      }
+
+      return false;
+    }, [chipCharacterLimit, chipTruncationLocation, displayName, isTruncated]);
+
+    useEffect(() => {
+      if (isFocused) {
+        buttonRef?.current?.focus();
+      }
+    }, [forwardedRef, isFocused]);
+
+    const handleKeyDown = (e: React.KeyboardEvent) => {
+      if (
+        e.keyCode === 8 || // Delete
+        e.keyCode === keyMap.Enter ||
+        e.keyCode === keyMap.Space
+      ) {
+        onRemove();
+      }
+    };
+
+    const dataProp = createDataProp('combobox-chip');
+
+    return (
+      <span
+        data-test-id="lg-combobox-chip"
+        {...dataProp.prop}
+        ref={forwardedRef}
+        className={chipWrapperStyle({ darkMode, size })}
```

Comment:
It would be rad if you could click on a chip (outside of the "x"), and it would select the chip like you can with the arrow keys.